app/routes/home.tsx

```
@@ -0,0 +1,46 @@
+import type { Route } from './+types/home'
+import { Header } from '../components/Header'
+import { Footer } from '../components/Footer'
+import { BlogCard } from '../components/BlogCard'
+import { getAllPosts } from '../data/posts.server'
+
+export function meta({}: Route.MetaArgs) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)